components/StoreLocatorModal.vue

```diff
@@ -75,7 +79,7 @@
   </SfModal>
 </template>
 <script lang="ts">
-import { SfModal, SfSearchBar } from "@storefront-ui/vue"
+import { SfModal, SfSearchBar, SfBar } from "@storefront-ui/vue"
 import { computed, ref } from "@nuxtjs/composition-api"
 import KiboStoreDetails from "@/components/KiboStoreDetails.vue"
```

Comment:
KiboStoreDetails component should be autoimported

Reply:
Changed